docs/symantec-solution-pack-playbook-doc.md

```diff
@@ -1,11 +1,11 @@
 # FortiSOAR Symantec Solutions Solution Pack – Out-of-the box Playbooks Collections
 
-The FortiSOAR™ Symantec Solutions Solution Pack (solution-pack-symantec-solutions). This solution pack enables users to experience the power and capability of FortiSOAR™ incident response for Symantec Solutions. This solution pack provides capabilities that show case the Symantec Cloud SOC use case.
+The FortiSOAR™ Symantec Solutions Solution Pack (solution-pack-symantec-solutions). This solution pack enables users to experience the power and capability of FortiSOAR™ incident response for Symantec Solutions. This solution pack provides capabilities that showcase the Symantec Cloud SOC use case.
```

Comment:
Please change the file name to solution-pack-guide.md and make it consistent with all other .md files